### PR TITLE
[hidden regions] Code Cleanup

### PR DESCRIPTION
**Pull request by meysholdt**
Signed-off-by: Moritz Eysholdt <moritz.eysholdt@typefox.io>

### CODE REVIEW

File: org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/HiddenRegionPartAssociator.java [hunk 1]
```diff
@@ -25,12 +25,15 @@
...
 		if (!region.isMultiline()) {
+			// when there is no line-wrap between two semantic tokens, assume that comments belong to the CONTAINER
+			// rather than one of the surrounding semantic tokens.
 			for (IHiddenRegionPart p : parts) {
 				handler.accept(p, HiddenRegionPartAssociation.CONTAINER);
 			}
 		} else {
 			int i = 0;
 			if (region.getPreviousSemanticRegion() != null) {
+				// collect whitespace and comments from the same line as belonging to the PREVIOUS semantic token
```
Comment (cdietrich) [inline org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/HiddenRegionPartAssociator.java]: typo? are?

Comment (meysholdt) [reply org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/HiddenRegionPartAssociator.java]: no, both whitespace AND comments are collected.

Comment (cdietrich) [reply org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/HiddenRegionPartAssociator.java]: no i means `as` sounds wired in the sentence

Comment (meysholdt) [reply org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/HiddenRegionPartAssociator.java]: I see. Same as in `they're treated as...`

**Pull request merged**